src/components/atoms/canvas.tsx

```
@@ -152,229 +26,225 @@ export interface Coordinates {
+                            x: clickCoordinates.x,
+                            y: clickCoordinates.y,
+                        };
+                        socket.emit('input:mousemove', {
+                            x: clickCoordinates.x,
+                            y: clickCoordinates.y,
+                        });
+                        notifyLastAction('move');
+                    }
+                    break;
+                case 'wheel':
+                    const wheelEvent = event as WheelEvent;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)